exercises/practice/missing-number/missing_number_test.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+# from missing_number import missing_number
+from missing_number_example import missing_number
+
+class MyTestCase(unittest.TestCase):
+    def test50(self):
+        """Several tests with numbers in the range of 1 to 50."""
+        N = 50
+        exclude = [1, 50, 24, 33, 19, 2, 49]
+        for ex in exclude:
```

Comment:
Do you think it would be beneficial to the user if we had separate test cases instead of using a loop so that they can see which test fails?

Reply:
Added a failure message.